src/gtc/gtcpp/gtcpp_codegen.py

```diff
@@ -103,14 +103,27 @@ def visit_NativeFunction(self, func: NativeFunction, **kwargs: Any) -> str:
                 NativeFunction.MIN: "gridtools::math::min",
                 NativeFunction.MAX: "gridtools::math::max",
                 NativeFunction.MOD: "gridtools::math::fmod",
+                NativeFunction.SIN: "std::sin",
+                NativeFunction.COS: "std::cos",
+                NativeFunction.TAN: "std::tan",
+                NativeFunction.ARCSIN: "std::asin",
+                NativeFunction.ARCCOS: "std::acos",
+                NativeFunction.ARCTAN: "std::atan",
```

Comment:
Question: aren't there `gridtools::math::...` equivalent functions instead of `std::`? Or, if there are not benefits using `gridools:math::...` functions because all GridTools backends accept  `std::` math functions, wouldn't it be better to use the same namespace for all the functions here?

Reply:
I checked with all recent CUDA versions and the GT people, looks like we can indeed use `std::` stuff everywhere.